pkg/cmd/util.go

```diff
@@ -0,0 +1,196 @@
+package cmd
+
+import (
+	"bufio"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"go.uber.org/zap"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/kube-openapi/pkg/validation/spec"
+)
+
+const extensionGVK = "x-kubernetes-group-version-kind"
+
+type util struct {
+	swagger *spec.Swagger
+	logger  *zap.SugaredLogger
+}
```

Comment:
nit: I don't think this type is necessary and those function were fine as function imo

Reply:
Done.

Reply:
Removed struct.